corehq/apps/app_manager/views/forms.py

```diff
@@ -546,12 +547,19 @@ def get_form_view_context_and_template(request, domain, form, langs, messages=me
         form.get_unique_id()
         app.save()
 
+    allow_usercase = (domain_has_privilege(request.domain, privileges.USER_CASE)
```

Comment:
i don't have much context but am just curious if you also need to check if usercase is enabled for the project here, or if that doesn't matter